docker/sf-gpu.Dockerfile

```diff
@@ -1,3 +1,28 @@
 FROM tensorflow/tensorflow:2.10.1-gpu
-
-RUN pip install -U secretflow
+#install secretflow
+#add the mirror source of doubanio to accelerate installation of SecretFlow and accelerate the building of images
+#if not, the building of images are prone to fail very much 
+RUN pip install -U secretflow  -i  https://pypi.doubanio.com/simple/
```

Comment:
I already guessed that you might ask me to do this, however, as I emphasized in the comments, **removing these would make the image build particularly slow or even fail**. So, can you think twice about keeping them? because keeping these can make the image build successful in one step. @anakinxc

Reply:
Considering that both the SecretFlow's dependecies packages such as PyTorch and Tensorflow  and NVIDIA toolkits packages are very large files, which make the installation is very slow and easy to fail, it is especially necessary to use mirror site to accelerate the installation.

Reply:
Hi @Mingbo-Lee 

Consider this is an opensource project that can be used by different people from different regions, please do not make any assumption on network condition.

Reply:
I have modified the Dockerfile as you suggested. Please TBR. @anakinxc